examples/Default.example

```diff
@@ -3,41 +3,66 @@ class Example extends React.Component {
   constructor(props) {
     super(props);
     cookie('accepts-cookies', '');
-    this.state = {
```

Comment:
@veej done.
Can you please re-enable the `dismissOnScroll` and test it again on master?